core/src/main/java/io/confluent/kafka/schemaregistry/rest/Main.java

```diff
@@ -23,6 +23,14 @@
 
 import io.confluent.rest.RestConfigException;
 
+/**
+ * @deprecated
+ * This duplicates code in {@link io.confluent.kafka.schemaregistry.rest.SchemaRegistryMain}
+ * for compatibility.
+ *
+ * See https://github.com/confluentinc/schema-registry/issues/194
+ */
+@Deprecated
 public class Main {
```

Comment:
I think this is exactly what we want in terms of annotation, but we should also remove all the duplicate code. I think just invoking `io.confluent.kafka.schemaregistry.rest.SchemaRegistryMain.main(args)` here works.